lib/CL/clGetPlatformIDs.c

```diff
@@ -37,7 +37,7 @@
 */
 #ifdef BUILD_ICD
 struct _cl_icd_dispatch pocl_dispatch = {
-  &POname(clGetPlatformIDs),
+  (void *)CL_ICD2_TAG_KHR,
```

Comment:
Pls add a comment explaining what the TAG does (so the code reader doesn't have to check the ICD2 spec).